### DISCUSSION TITLE
bluos losing sessionid

### DISCUSSION THREAD

**Discussion by talljhawkins** [tidal-music/discussions#59] [Report bug]
Hi I'm using the tidal connect app in android and also the bluos app which uses some tidal API underneath ( I don't know what they do).
Every now and then I will switch on my bluos app and it will have lost the sessionID.
can someone tell me how the session Id works please ? They only use username and password to login - is that still supported? What ttl is on the sessionID - forever? Does it have to be refreshed? If I don't refresh it (turn on my bluos presumably) then does it timeout after a while ??

many thanks !
John.

**Comment by stefanmoro** [comment]
Hey @talljhawkins ,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API, open source projects etc).

As this question relates to an existing product feature, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
